recipes/nmos-cpp/all/conanfile.py

```diff
@@ -249,5 +246,6 @@ def _register_components():
 
         # add nmos-cpp-registry and nmos-cpp-node to the path
         bin_path = os.path.join(self.package_folder, bindir)
-        self.output.info("Appending PATH environment variable: {}".format(bin_path))
+        self.output.info(f"Appending PATH environment variable: {bin_path}")
+        self.runenv_info.prepend_path("PATH", bin_path)
```

Comment:
```suggestion
```

Not needed. Both runenv and buildenv automatically appends PATH:

https://docs.conan.io/en/latest/reference/conanfile/tools/env/virtualbuildenv.html

Reply:
I'm still having a hard time making the `test_package` work reliably. Obviously it all works in c3i... but locally on a Windows box with conan 1.53.0 or 1.54.0, the test package fails during `conan create`, but succeeds when `conan test` is run standalone, apparently because the injected environment PATH is different.

I.e.

```
> conan create recipes/nmos-cpp/all/conanfile.py cci.20221111@ -pr:h default -pr:b default
[...]
nmos-cpp/cci.20221116 (test package): Running test()
ERROR: nmos-cpp/cci.20221116 (test package): Error in test() method, line 34
        registry_path = shutil.which("nmos-cpp-registry", path=os.pathsep.join(self.env["PATH"]))
        KeyError: 'PATH'
```

And then this succeeds:
```
conan test recipes/nmos-cpp/all/test_package nmos-cpp/cci.20221116@
```

I wrote up a previous comment about this (https://github.com/conan-io/conan-center-index/pull/14152#issuecomment-1313783504) and both @samuel-emrys and @prince-chrismc made helpful suggestions on Slack but I'm still not sure what the right thing to do is, maybe in the recipe `package_info` here or in the test package, to make this work.